source/PyMaterialX/PyMaterialXGenShader/PyShaderStage.cpp

```diff
@@ -0,0 +1,47 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <PyMaterialX/PyMaterialX.h>
+
+#include <MaterialXGenShader/ShaderStage.h>
+
+namespace py = pybind11;
+namespace mx = MaterialX;
+
+void bindPyShaderStage(py::module& mod)
+{
+    mod.attr("PIXEL_STAGE") = &mx::Stage::PIXEL;
+
+    py::class_<mx::VariableBlock, mx::VariableBlockPtr>(mod, "VariableBlock")
+        .def(py::init<const std::string&, const std::string&>())
+        .def("getName", &mx::VariableBlock::getName)
+        .def("getInstance", &mx::VariableBlock::getInstance)
+        .def("empty", &mx::VariableBlock::empty)
+        .def("size", &mx::VariableBlock::size)
+        .def("find", static_cast<mx::ShaderPort* (mx::VariableBlock::*)(const std::string&)>(&mx::VariableBlock::find))
+        .def("add", static_cast<mx::ShaderPort* (mx::VariableBlock::*)(const mx::TypeDesc*, const std::string&, mx::ValuePtr)>(&mx::VariableBlock::add))
```

Comment:
For now we can probably leave this class read-only from the Python side, so we can remove the two "add" methods here.